openmetadata-ui/src/main/resources/ui/src/locale/languages/mr-in.json

```diff
@@ -533,6 +533,7 @@
     "explore-now": "आता अन्वेषण करा",
     "export": "निर्यात करा",
     "export-entity": "{{entity}} निर्यात करा",
+    "export-type": "निर्यात प्रकार",
```

Comment:
@aniketkatkar97 can you please validate this?

Reply:
It's correct